partner_used_dealer/views/res_partner_views.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">res.partner.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='function']" position="after">
+                <field name="date_of_birth" widget="date" />
+            </xpath>
+            <xpath expr="//field[@name='website']" position="after">
+                <field name="occupation_id" />
+                <field name="identification_type_id" />
+                <field name="identification_number" />
+            </xpath>
```

Comment:
I prefer to have the added fields in one place (perhaps add a tab in the partner form to keep all the added fields). We may later check with the customer on this.